src/app/shared/model/NodeSpecChange.ts

```diff
@@ -11,13 +11,12 @@
 
 import {
   CentosSpec,
-  ContainerLinuxSpec,
   NodeCloudSpec,
   NodeSpec,
   RHELSpec,
   SLESSpec,
   UbuntuSpec,
-  FlatcarSpec,
+  FlatcarSpec
```

Comment:
Please keep the commas unless a linter says otherwise. :)

Reply:
Linter is not happy :slightly_smiling_face: : https://public-prow.loodse.com/view/gs/prow-dev-public-data/pr-logs/pull/kubermatic_dashboard/3027/pre-dashboard-check-static/1356285079453700096#1:build-log.txt%3A348

If you've managed to fix our docker scripts then `npm run fix` should fix most of the issues.